wait/host_port.go

```diff
@@ -39,6 +41,12 @@ func ForListeningPort(port nat.Port) *HostPortStrategy {
 	return NewHostPortStrategy(port)
 }
 
+// ForExposedPort constructs an exposed port strategy. Alias for `NewHostPortStrategy("")`.
+// This strategy waits port exposed in Docker container.
```

Comment:
It's always the first port, right? https://github.com/testcontainers/testcontainers-go/pull/468/files#diff-ad6cdf7fcae968ad7307fe3e8717bcbc7e43c461424e76ca3953cd215ac9395bR77
```suggestion
// This strategy waits for the first port exposed in the Docker container.
```